src/Avalonia.Themes.Default/Label.xaml

```diff
@@ -0,0 +1,16 @@
+<Style xmlns="https://github.com/avaloniaui" Selector="Label">
```

Comment:
Template for the fluent theme is missing (you can copy-paste this one)

Reply:
PR targets 0.9.12 for some reason

Reply:
> Template for the fluent theme is missing (you can copy-paste this one)

Where is it? can't find in Avalonia Repo... :( maybe i'm blind???? ;(

Reply:
https://github.com/AvaloniaUI/Avalonia/tree/master/src/Avalonia.Themes.Fluent

Reply:
found it. it was added after 0.9.12